src/jit/gentree.h

```diff
@@ -1459,9 +1454,10 @@ struct GenTree
             case GT_INTRINSIC:
             case GT_LEA:
             case GT_STOREIND:
-            case GT_INITBLK:
-            case GT_COPYBLK:
-            case GT_COPYOBJ:
+            case GT_BLK:
+            case GT_OBJ:
```

Comment:
I think all of the unary ops have a null for op2.


---
In reply to: [77283180](https://github.com/dotnet/coreclr/pull/7027#discussion_r77283180) [](ancestors = 77283180,77282926)